03_map_set.js

```diff
@@ -0,0 +1,27 @@
+const arr = [ { id: 1 }, { id: 2 }, { id: 3 }, { id: 1 }];
+
+let map = new Map;
+let set = new Set;
+
+function deleteDuplication (arr) {
+  for (let i = 0; i < arr.length; i++){
```

Comment:
You have 3 for cycles. This is not optimized. Use at least 1 (not more). Use collection that you have learned last lecture.

Reply:
done